tests/models/whisper/test_modeling_whisper.py

```diff
@@ -3004,12 +3042,21 @@ def test_whisper_longform_multi_batch_hard(self):
         audio = ds[:num_samples]["audio"]
         audios = [x["array"] for x in audio]
 
+        gen_kwargs = {
+            "return_timestamps": True,
+            "no_speech_threshold": 0.6, # necessary to trigger no speech detection
+            "temperature": (0.0, 0.2, 0.4, 0.6, 0.8, 1.0),
+            "compression_ratio_threshold": 1.35,
+            "condition_on_prev_tokens": False,
+            "logprob_threshold": -2.0, # necessary to avoid triggering temp fallback that will introduce randomness since we are comparing to openai EXTECTED_TEXT
+        }
```

Comment:
Some of these are already used by default, let's remove them to improve readability

Reply:
All the ones with numerical values here are not set by default (not in generate and not also in the model's generation_config.json). Concerning `return_timestamps` and `condition_on_prev`, I find it clearer to have them explicitly mentioned.